hydradx/other tests/test_price_manipulations.py

```diff
@@ -0,0 +1,479 @@
+import copy
```

Comment:
I am nervous about a folder name with a space in it, but maybe that is okay these days... I'd prefer other_tests